src/Aspire.Dashboard/Components/Controls/ResourceActions.razor.cs

```diff
@@ -49,7 +49,7 @@ protected override void OnParametersSet()
             OnClick = OnConsoleLogs.InvokeAsync
         });
 
-        var menuCommands = Commands.Where(c => !c.IsHighlighted).ToList();
+        var menuCommands = Commands.Where(c => !c.IsHighlighted && c.State != CommandViewModelState.Hidden).ToList();
```

Comment:
Should the context menu show all commands regardless of whether they are highlighted or not?

Reply:
Dunno. It's easy to change.